src/pages/Detail/Details.js

```diff
@@ -0,0 +1,118 @@
+import CloseButton from 'react-bootstrap/CloseButton';
+import ListGroup from 'react-bootstrap/ListGroup';
+import React, { useEffect } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import './details.css';
+import { allCoins } from '../../redux/home-reducer';
+
+function Details() {
+  const getName = useParams();
+  const { name } = getName;
+  const Coins = useSelector((state) => state.coin);
+  const send = useDispatch();
+  useEffect(() => {
+    send(allCoins());
+  }, [send]);
+
+  const filtere = Coins.filter((item) => item.name === name);
+  return (
+    <div className="detail-wrapper">
+      {
+          filtere.map((item) => (
+            <div key={item.id}>
+              <div className="header">
+                <div>
+                  <Link to="/" type="button"><CloseButton /></Link>
+                  <img className="images" src={item.icon} alt={item.name} />
+                </div>
+                <div className="d-flex data">
+                  <h1>{item.name}</h1>
+                  <p style={{ color: 'white' }}>
+                    Price : $
+                    {Math.round(item.price * 100) / 100 }
+                  </p>
+                </div>
+              </div>
+              <p className="satas">Stats Of Coin</p>
+              <ListGroup as="ol" className="items-list">
+                <ListGroup.Item as="li" className="list">
+                  <p>Rank :</p>
+                  <p>{item.rank}</p>
+                </ListGroup.Item>
+                <ListGroup.Item as="li" className="list">
+                  <p>Price :</p>
+                  <p>
+                    $
+                    {Math.round(item.price * 100) / 100 }
+                  </p>
+                </ListGroup.Item>
+                <ListGroup.Item as="li" className="list">
+                  <p>Price in BTC :</p>
+                  <p>
+                    $
+                    {Math.round(item.priceBtc * 100) / 100 }
+                  </p>
+                </ListGroup.Item>
+                <ListGroup.Item as="li" className="list">
+                  <p>Volume :</p>
+                  <p>
+                    $
+                    {Math.round(item.volume * 100) / 100 }
+                  </p>
+                </ListGroup.Item>
+                <ListGroup.Item as="li" className="list">
+                  <p>MarketCap :</p>
+                  <p>
+                    $
+                    {Math.round(item.marketCap * 100) / 100 }
+                  </p>
+                </ListGroup.Item>
+                <ListGroup.Item as="li" className="list">
+                  <p>Available Supply :</p>
+                  <p>
+                    $
+                    {Math.round(item.availableSupply * 100) / 100 }
+                  </p>
+                </ListGroup.Item>
+                <ListGroup.Item as="li" className="list">
```

Comment:
- [ ] As mentioned by the previous reviewer the **Details** page design to match the one from the template, I suggest you improve it again by adding the `LeftArrowIcon` at the `header` like in the image below(I can suggest **[react-icons](https://react-icons.github.io/react-icons/)**):

| Template design | Your Design |
| ----------------------- | -------------------- |
| ![Screenshot from 2023-02-10 21-59-28](https://user-images.githubusercontent.com/83042742/218186809-b8f8e7e7-b63c-40a3-adda-bcfdffe3b51d.png) | ![Screenshot from 2023-02-10 22-00-03](https://user-images.githubusercontent.com/83042742/218186864-cdf95024-9bc1-47dd-a300-20aea5dd5153.png) |